src/Module/Admin/Site.php

```diff
@@ -266,6 +267,7 @@ protected function post(array $request = [])
 		DI::config()->set('system', 'block_extended_register', !$enable_multi_reg);
 		DI::config()->set('system', 'no_openid'              , !$enable_openid);
 		DI::config()->set('system', 'no_regfullname'         , !$enable_regfullname);
+		DI::config()->set('system', 'register_notification'  , $register_notification);
```

Comment:
Please add this new config key to `static/settings.config.php` as well.

Reply:
Done